### PR TITLE
added database-week1 homework

### PR DESCRIPTION
**Pull request by shravanipagilla**

### CODE REVIEW

File: databases/week1/database-week1.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+DROP DATABASE IF EXISTS hyf_db;
+
+CREATE DATABASE IF NOT EXISTS hyf_db;
+
+USE hyf_db;
+
+-- 1.Find out how many tasks are in the task table
+SELECT *
```
Comment (ReenuMary) [inline databases/week1/database-week1.sql]: To get number of tasks use Count(*)

File: databases/week1/database-week1.sql [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+CREATE DATABASE IF NOT EXISTS hyf_db;
+
+USE hyf_db;
+
+-- 1.Find out how many tasks are in the task table
+SELECT * 
+FROM `hyf_db`.`task`;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT due_date
```
Comment (ReenuMary) [inline databases/week1/database-week1.sql]: Here also you can use count(*) for getting the number of tasks

File: databases/week1/database-week1.sql [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT due_date 
+FROM `hyf_db`.`task`
+WHERE due_date IS  NULL;
+
+-- 3.Find all the tasks that are marked as done
+SELECT `title`
+FROM `hyf_db`.`task`
+WHERE `task`.`status_id`= '3'
```
Comment (ReenuMary) [inline databases/week1/database-week1.sql]: can also do a  join  with the status table where the status.name='Done'

File: databases/week1/database-week1.sql [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+WHERE `task`.`description` like '%database%' OR
+      `task`.`title` like '%database%';
+-- 8.Get the title and status (as text) of all tasks
+SELECT `title`,`name`
+FROM `hyf_db`.`task`, `hyf_db`.`status`
+WHERE `task`.`status_id`= `status`.`id`
+
+
+-- 9.Get the name of each status, along with a count of how many tasks have that status
+SELECT COUNT(name)
```
Comment (ReenuMary) [inline databases/week1/database-week1.sql:54]: you need to select status.name also , to get the number of tasks with the status, use group by

File: databases/week1/database-week1.sql (hunk 1 above)
Comment (shravanipagilla) [reply databases/week1/database-week1.sql]: Hej @ReenuMary . thanks for comment on my homework , after pushing i  realized  that. i will do as comment thank you ....:)

**Review by Elia-Git — APPROVED** [review]
Ship it pls.

**Comment by shravanipagilla** [comment]
Thank you for review on my code ... :)

**Pull request merged**